### PR TITLE
Slow movement for sim, fix bug of tag, add max number of tag to 32

### PR DESCRIPTION
**Pull request by xiejiangzhi**
Holding ctrl to slowly movement for Sim
Check null for findTag
Increment max number of tags to 32
Fix newWorld error if tags > MAX_TAGS

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@
 bin
 build*
 *.*.h
+.vscode
```
Comment (bjornbytes) [inline .gitignore]: This should go in your local `.git/info/exclude` (or an excludesFile like `~/.gitignore`)

File: src/modules/physics/physics.c [hunk 2]
```diff
@@ -171,7 +172,7 @@ World* lovrWorldCreate(float xg, float yg, float zg, bool allowSleep, const char
     world->tags[i] = malloc(size);
     memcpy(world->tags[i], tags[i], size);
   }
-  memset(world->masks, 0xff, sizeof(world->masks));
+  memset(world->masks, 0xffff, sizeof(world->masks));
```
Comment (bjornbytes) [inline src/modules/physics/physics.c]: The clear value for `memset` is a single byte, so this does not need to be changed from `0xff`.

File: src/api/l_physics.c [hunk 3]
```diff
@@ -25,10 +25,13 @@ static int l_lovrPhysicsNewWorld(lua_State* L) {
...
   float zg = luax_optfloat(L, 3, 0.f);
   bool allowSleep = lua_gettop(L) < 4 || lua_toboolean(L, 4);
-  const char* tags[16];
+  const char* tags[MAX_TAGS];
   int tagCount;
   if (lua_type(L, 5) == LUA_TTABLE) {
     tagCount = luax_len(L, 5);
+    if (tagCount > MAX_TAGS) {
+      return luaL_error(L, "World max number of tags is 32");
+    }
```
Comment (bjornbytes) [inline src/api/l_physics.c]: Minor suggestion

```suggestion
    lovrCheck(tagCount <= MAX_TAGS, "Max number of world tags is %d", MAX_TAGS);
```

**Comment by xiejiangzhi** [comment]
Fixed

**Comment by bjornbytes** [comment]
Thanks for the fixes!

**Pull request merged**